mos-platform/c64/CMakeLists.txt

```diff
@@ -1,30 +1,10 @@
-platform(c64 COMPLETE HOSTED PARENT common)
```

Comment:
I think we still need the HOSTED tag here, since the target has a real operating system (although right now that's just putchar, getchar, and malloc).

Reply:
ok, fixed that.